compiler/symbol.go

```diff
@@ -151,8 +153,13 @@ func (c *compilerContext) getFunction(fn *ssa.Function) llvm.Value {
 	if info.exported {
 		// Set the wasm-import-module attribute if the function's module is set.
 		if info.module != "" {
+
+			// We need to add the wasm-import-module and the wasm-import-name
 			wasmImportModuleAttr := c.ctx.CreateStringAttribute("wasm-import-module", info.module)
 			llvmFn.AddFunctionAttr(wasmImportModuleAttr)
+
+			wasmImportNameAttr := c.ctx.CreateStringAttribute("wasm-import-name", info.importName)
+			llvmFn.AddFunctionAttr(wasmImportNameAttr)
```

Comment:
Perhaps this needs an extra check that `info.importName` is not unset (an empty string)?